test-commons/src/main/java/com/sonarsource/checks/verifier/LineIssues.java

```diff
@@ -167,14 +167,24 @@ private void appendLocations(StringBuilder out) {
       Map<Integer, PerLineLocationWriter> writerPerLine = new TreeMap<>();
       for (PreciseLocation location : locations) {
         writerPerLine.computeIfAbsent(
-          location.range.line, key -> new PerLineLocationWriter(String.format("%03d: ", key), testFile.lineWithoutComment(key))).add(location);
+          location.range.line, key -> new PerLineLocationWriter(String.format("%03d: ", key), reportLineAt(key))).add(location);
```

Comment:
@alban-auzeill can we use something more explicit than `line == 0` to spot file-issues?